index.css

```diff
@@ -226,135 +236,139 @@ main {
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin: 0 24px 0 24px;
+  display: flex;
+  flex-direction: column;
 }
 
 .professional__project-container {
-  display: flex;
-  flex-direction: column;
-  width: 327px;
-  margin-top: 80px;
-  padding: 10px;
+  padding: 25% 5% 0 5%;
 }
 
 .professional__project-title {
-  width: 295px;
-  font-size: 400;
-  font-size: 32px;
   font-family: 'Crete Round', sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
   color: #fff;
 }
 
 .professional__project-detail {
   font-family: 'Inter', sans-serif;
-  width: 295px;
   font-size: 16px;
+  font-style: normal;
   color: #fff;
-  line-height: 24px;
+  padding: 10% 10% 0 0;
 }
 
-.profession__project-butons ul li a {
-  text-decoration: none;
-  padding-right: 20px;
-  justify-content: space-around;
+.profession__project-butons {
+  padding: 20px 0 0 0;
 }
 
-.profession__project-butons a button {
-  padding: 10px;
-  background: rgba(255, 255, 255, 0.24);
-  border: none;
-  color: #fff;
-  width: 80px;
-  height: 40px;
+.professional__button-select {
   font-family: 'Inter', sans-serif;
-  font-size: 15px;
-  line-height: 20px;
   font-weight: 400;
+  font-size: 16px;
+  border: none;
+  margin: 0 10 0 0;
+  padding: 10px 10px 10px 0;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
 }
 
 .professional__see-project {
-  margin-top: 10px;
   display: flex;
-  width: 327px;
-  height: 48px;
-  background: #ff6b00;
-  color: #fff;
-  align-items: center;
+  background: #f55800;
+  border: 1px solid #f55800;
   justify-content: center;
+  margin-top: 50px;
+  color: #fff;
+}
+
+.professional__see-project:hover {
+  background: #fff5e1;
+  color: #f55800;
+  font-weight: 400;
+}
+
+.professional__project-link {
   font-family: 'Inter', sans-serif;
-  text-align: center;
-  font-stretch: 17px;
+  font-size: 17px;
+  font-weight: 700;
+  font-style: normal;
   line-height: 24px;
-  letter-spacing: 0.03em;
-  border: 1px solid #ff6b00;
+  text-decoration: none;
+  color: #fff;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  padding: 10px;
+  width: 100%;
 }
 
-.professional__see-project:hover {
+.professional__project-link:hover {
   background: #fff5e1;
   color: #f55800;
   font-weight: 400;
 }
 
 /* about me page  */
 .aboutme {
-  width: 375;
-  height: 750px;
   display: flex;
   flex-direction: column;
   background: #1c1a19;
+  background-image: url(../icons_copy/amBackground.svg);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  padding: 0 0 40% 0;
+  border-bottom: 2px solid #c9cfcb;
 }
 
 .aboutme__container {
   display: flex;
   flex-direction: column;
-  align-items: center;
+  justify-content: flex-start;
+  padding: 150px 24px 0 24px;
 }
 
 .aboutme__title {
-  width: 327px;
   font-family: 'Crete Round', sans-serif;
-  font-size: 100px;
-  letter-spacing: 0.37;
+  font-size: 72px;
```

Comment:
- [ ] Kindly fix the button padding on this section to match the Figma design
- [ ] Kindly fix the size of your text, since this is very big than the one on Figma

![image](https://user-images.githubusercontent.com/93630700/213793071-d6d9f30e-e5e6-4b80-95b5-500254d7944b.png)